### ISSUE TITLE
Arrow alignment check shouldn't span over several resources

### ISSUE THREAD

**Issue by ghost** [rodjek/puppet-lint#78]
This shouldn't give an error but does:

```
file { 'foo': ensure => file }
package { 'bar': ensure => present }
```
